DATABASE_MANAGEMENT.md

```diff
@@ -70,4 +70,55 @@ A single database is restricted to 2 billion nodes (also, see [BaseX Statistics]
 A node in this case is an XML node like an element, attribute, text, etc.
 
 ### Sources
-http://www.adamretter.org.uk/presentations/restxq_mugl_20120308.pdf
\ No newline at end of file
+http://www.adamretter.org.uk/presentations/restxq_mugl_20120308.pdf
+
+## Provenance Overview
```

Comment:
Where do you want to store de prov information in the database? In a seperate database?